djvm/src/main/java/sandbox/java/lang/ref/Reference.java

```diff
@@ -0,0 +1,31 @@
+package sandbox.java.lang.ref;
+
+@SuppressWarnings({"WeakerAccess", "unused"})
+public abstract class Reference<T> extends sandbox.java.lang.Object {
```

Comment:
Why do you need to reimplement `Reference`?

Reply:
Because it's the base class of `SoftReference`, which is used when loading Java `Locale` objects. The original `Reference` implementation does "unsafe things" with `Thread`, but switching it with my own version allows the `Locale`s to load correctly.